lib/Recipient.php

```diff
@@ -6,6 +6,22 @@
  * Class Recipient
  *
  * @package Stripe
+ *
+ * @property string $id
+ * @property string $object
+ * @property mixed $active_account
+ * @property Collection $cards
+ * @property int $created
+ * @property string $default_card
+ * @property string $description
+ * @property string $email
+ * @property bool $livemode
+ * @property StripeObject $metadata
+ * @property string $migrated_to
+ * @property string $name
+ * @property string $rolled_back_from
+ * @property string $type
+ * @property bool $verified
```

Comment:
I wonder if we should pull `verified`? I think it's technically supposed to be undocumented.